### ISSUE TITLE
Role of the controls inside "Styles" and "Colors" toolbar of CK Editor is inappropriate

### ISSUE THREAD

**Issue by sagar17sep** [ckeditor/ckeditor-dev#1954]
**BUG**

**Provide detailed reproduction steps (if any)**
1. Navigate to CK Editor toolbar using Alt + F10.
2. Open Inspect Tool. 
3. Navigate to controls inside "Styles" and "Colors" toolbar.

**Expected result**
Role of the controls inside "Styles" and "Colors" toolbar - "Formatting styles", "Paragraph format", "Text color" and "Paragraph Color" should be combobox since it contains list items after expanding.

**Actual result**
Role of the controls inside "Styles" and "Colors" toolbar - "Formatting styles", "Paragraph format", "Text color" and "Paragraph Color" is inappropriate. Their role is button instead inspite of the list contained after expanding.

**Other details**
Browser: Edge 41.16299.248.0
OS: Windows 10
CKEditor version: 4.6.2
Installed CKEditor plugins:

**Comment by sagar17sep** [comment]
Could you please look to the above mentioned issue on priority.

**Comment by sagar17sep** [comment]
![image](https://user-images.githubusercontent.com/38970470/40957397-a327be8a-68b2-11e8-90db-476adee112f0.png)

Could you please look to the above mentioned issue on priority.

**Comment by mlewand** [comment]
The fact that buttons lead to a dropdown which contains a listbox is denoted by a `aria-haspopup="true"` attribute.

The content of opened dropdown contains a listbox role element (and `[role=option]` for its children) which is perfectly valid use, [conforming to standards](https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html).

So there is a rich markup, properly conveying content type, so screen readers or any assistive technology conforming WAI-ARIA should have no issue on picking it up.

During the review of this plugin we noticed that WAI-ARIA 1.1 improved `aria-haspopup` value dictionary, allowing us to better specify type of UI component controlled by the button. It's a different, but related matter, so we created a separate issue for that in #2072.
